Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,39 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        static void CheckArgument(Object obj)
```

Comment:
Specify access modifiers. From the coding guidelines

> DO use the C# keyword rather than the BCL name when specifying a data type (e.g., string rather than String).

```suggestion
        private static void CheckArgument(object obj)
```